packages/eslint-plugin/tests/rules/non-nullable-type-assertion-style.test.ts

```diff
@@ -199,6 +199,42 @@ declare const x: T;
 const y = x!;
       `,
     },
+    {
```

Comment:
#6887 - could you split this in two please?

Reply:
Thank you for review, I've splitted it!